### ISSUE TITLE
Stuck in the final boss chamber after the fight

### ISSUE THREAD

**Issue by TMaluszynski** [The-Gate-Project/IWD1_EET#19]
After beating the final boss of Icewind Dale 1 the final cinematic played and my party was stuck inside the chamber with no way to get out other than console commands.

No errors showed up in logs.

Installed mods (in order):
-EET
-IWD1
-IWD2
-Cowled Menace
-NWNforBG2
-EEeX
-Wild Mage
-subrace
-d2-weapon
-bp-bgt-worldmap
-EET Tweaks
-IWD end
-EET end

**Comment by 11jo** [comment]
Hello, it's possible that a variable isn't set properly.

Have you completed TotLM and HoW before ?

- I just did a quick test, once Maladernoth send me to Easthaven, CluaConsole to id1104.are then going through the mirror.
After the final fight transition work fine it seem.

- But if I go directly to IWD id1104.are threw CluaConsole the transition won't work.

**Comment by TMaluszynski** [comment]
@11jo no, only the main campaign, I didn't go for HoW beyond talking to the shaman in Kuldahar (and since TotLM is started from HoW hub area neither did I go for that one).

Interestingly enough when I consoled my way back to Kuldahar and talked to the shaman the triggers seemed to think I got transported to Lonelywood, so the only thing missing is the area transition.

**Comment by TMaluszynski** [comment]
I traveled via Maladernoth, so not sure what happened here. Possibly some mod incompatibility? My list is quite long.

**Comment by 11jo** [comment]
But did you do the extensions ?

Your list above is not that long and I don't think anything could interfer.

You can try to enter this... C:Eval('StartCutScene("IDTOEET")') ... in CluaConsole to launch the transition.

**Comment by TMaluszynski** [comment]
I really don't want to do the final fight again lol.

**Comment by 11jo** [comment]
I understand and there is no warranty that the transition will trigger this time. But don't forget "**Ctrl-Y**" you can end the fight in one click.

**Comment by lukas0x1** [comment]
Hey! Sorry for the late response, I was a bit preoccupied. I did another mod install just not and am still facing the same issue. C:Eval('StartCutScene("IDTOEET")') heals me and then gets me stuck in a black space but C:Eval('StartCutScene("IDTOEET1")') did get me back to ulgoth. It seems like there's something wrong with the triggers? Could this be a bp-bgt-worldmap issue?

**Comment by lukas0x1** [comment]
Does IDTOEET1 clean up all the IWD related globals? I don't mind having to use the console as long as my save isn't borked in any way afterwards.

**Comment by lukas0x1** [comment]
I did my last mod install today and made sure to update iwd_1_eet iwd_2_eet and iwd_eet_end to the latest commit from their respective repositories. CLUAConsole:MoveToArea("ID0000") puts me in the area that C:Eval('StartCutScene("IDTOEET")') puts me in, but it doesn't trigger C:Eval('StartCutScene("IDTOEET1")')

**Comment by lukas0x1** [comment]
Which globals should I check to make sure that everything is fine? maybe I can just move to the area and then manually call IDTOEET

**Comment by lukas0x1** [comment]
I didn't do any of the IWD content this time around and just teleported to the mirror room after arriving in easthaven via maladernoth. When I posted my first comment two months ago I did however do a speedrun through iwd, but I think I also skipped HOW that time. Do you know the ID or name of the ICASARACHT room by any chance so that I can  confirm that everything works as intended if I play HOW? I'm planning to play HOW in my actual full playthrough but I tend to want to debug all issues before I start a proper run with a new modlist.

**Comment by lukas0x1** [comment]
Sweet! I re-run my mod install and let you know if it's resolved afterwards.

**Comment by lukas0x1** [comment]
After a re-install with the latest commit, It still doesn't seem to trigger any transitions after the final fight. Is there anything that it requires me to trigger beforehand that it would in a normal run? I didn't start HOW and just teleported to the final boss room again.

**Comment by lukas0x1** [comment]
I did start a new game and ran all the way through to maladernoth again, then teleported to id1105. I'll try a clean install with only iwd1, iwd2 and iwd_eet_end to check if it's a mod conflict.

**Comment by lukas0x1** [comment]
This is my relevant IWD section of my install order.

`~NWNFORBG/SETUP-NWNFORBG.TP2~ #1 #0 // NWN For BG: 0.99
~NWNFORBG/SETUP-NWNFORBG.TP2~ #1 #10 // NWN For BG - Worldmap select (EE-games only) -> Place NWN locations on BP-BGT Worldmap: 0.99
~NWNFORBG/SETUP-NWNFORBG.TP2~ #1 #20 // Install NWN as a dedicated campaign: 0.99
~IWD1_EET/SETUP-IWD1_EET.TP2~ #1 #0 // Use <CHARNAME> instead of <GABBER> etc.: 1.1
~IWD1_EET/SETUP-IWD1_EET.TP2~ #1 #1 // Icewind Dale: Heart of Winter in Baldur's Gate Enhanced Edition Trilogy: 1.1
~IWD2_EET/SETUP-IWD2_EET.TP2~ #1 #0 // Icewind Dale 2 in Baldur's Gate Enhanced Edition Trilogy: 1.1
~IWD_EET-PARTY-BANTER/SETUP-IWD_EET-PARTY-BANTER.TP2~ #0 #0 // Install IWDEET Party Banter: v1.0
~IWD_EET_END/SETUP-IWD_EET_END.TP2~ #1 #0 // IWD_EET_End: 1.0
~A7-TOTLM-BG2EE/A7-TOTLM-BG2EE.TP2~ #0 #0 // Trials of the Luremaster for Baldur's Gate: 4.3
~A7-TOTLM-BG2EE/A7-TOTLM-BG2EE.TP2~ #0 #20 // Rebalance weapons and equipment: 4.3
~A7-TOTLM-BG2EE/A7-TOTLM-BG2EE.TP2~ #0 #40 // Allow to trade with Hobart before starting the adventure: 4.3
~IWD_EET_INTEGRATION/IWD_EET_INTEGRATION.TP2~ #1 #100 // Add IWD Campaigns to the EET Menu: 1.2
~IWD_EET_INTEGRATION/IWD_EET_INTEGRATION.TP2~ #1 #150 // Add HoW access to BG2: 1.2
~IWD_EET_INTEGRATION/IWD_EET_INTEGRATION.TP2~ #1 #160 // Add IWD1 Easter Egg Content to BG2: 1.2
~IWD_EET_INTEGRATION/IWD_EET_INTEGRATION.TP2~ #1 #180 // Play NWN after the IWD1 Campaign: 1.2`

I could immagine that Play NWN after the IWD1 Campaign could be interfering with the transition sequence somehow since it's meant to trigger when played as a seperate campaign which I'm not. I'll check the scripts in a bit and try running without it to see if that does anything.

**Comment by lukas0x1** [comment]
I removed iwd_eet_integration entirely and that seems to have fixed the problem. I assume that it is either related to the NWN after IWD1 Campaign component or the HOW access in bg2 component.

**Comment by abalabokhin** [comment]
Yes, I am not very available lately, but can confirm that NWNForBG needs either second component or to have BPBGTWorldmap installed after it.
